tasks/compile-assets.mjs

```diff
@@ -14,7 +14,7 @@ export async function writeAsset (filePath, result) {
   const writeTasks = []
 
   // Files to write
-  const code = result.code || result.css?.toString()
+  const code = 'css' in result ? result.css : result.code
```

Comment:
Since Dart Sass landed in https://github.com/alphagov/govuk-frontend/pull/3164 `result.css` no longer needs `toString()`